src/components/app.js

```diff
@@ -74,9 +50,18 @@ class App extends React.Component {
             </div>
           </div>
           <div className='content-block'>
-            <div className='content-container'>
-              <Route exact path='/' component={Introduction} />
-              <Route path='/:componentKey' render={getComponent} />
+            <div className='content-container ds-l-row'>
+              <div className='ds-u-float--left ds-u-padding-right--6 ds-u-padding-top--2 ds-l-col--3'>
+                <ul className='ds-c-vertical-nav__subnav'>
+                  {LeftNav}
+                </ul>
+              </div>
+              <div className='ds-u-float--left ds-u-padding--1 ds-l-col--9'>
+                <div className='ds-u-measure--wide'>
+                  <Route exact path='/' component={Introduction} />
+                  <Route exact path='/:componentKey' render={ActiveComponent} />
```

Comment:
ah, seeing now how Routes is used below for navlink generation. What about a middle ground, inspired by the [React Router sidebar example](https://reacttraining.com/react-router/web/example/sidebar)?

```js
// routes.js
const routes = [
  { path: '/', exact: true, component: Introduction, linkText: 'Introduction' },
  { path: '/developer-preview', exact: false, component: DeveloperPreview, linkText: 'Getting a key' }
];

// App#render in app.js
  ...
  routes.map({path, exact, component}, index) => (
    <Route
      key={index}
      path={path}
      exact={exact}
      component={component}
      />
  )
  ...

// NavSection#render in leftnav.js
  ...
  routes.map({path, linkText}, index) => (
    <li key={index}>
      <NavLink to={path} activeClassName=''>{linkText}</NavLink>
    </li>
  )
  ...
```